### ISSUE TITLE
Ensure that root and metadata table can compact externally.

### ISSUE THREAD

**Issue by keith-turner** [apache/accumulo#3518]
After #3513 is merged there theoretically should be enough basic support for compacting the root and metadata tables via external compactions, but it probably does not work.  Need to work towards getting these compacting via external compactions.  Also need to explore if some basic ITs are needed to ensure that the root and metadata tablets are compacting going forward.

**Comment by dlmarion** [comment]
FWIW I remember during 2.1 testing that at some point internal compactions were broken.  IIRC @milleruntime configured his local instance to perform all compactions externally and it worked.

**Comment by keith-turner** [comment]
This is a duplicate of #3463.

**Comment by keith-turner** [comment]
> FWIW I remember during 2.1 testing that at some point internal compactions were broken. IIRC @milleruntime configured his local instance to perform all compactions externally and it worked.

Maybe if we change the default config, it will always work now that we have the mini changes.  Fingers crossed.
